### ISSUE TITLE
[Data Intake]: EcoMon Survey Strata

### ISSUE THREAD

**Issue by NMucci1** [jmhatch-NOAA/PEMAD-MDEB-Data_Dictionary#1]
### Data owner / manager

```markdown
Name: Harvey Walsh
Title: Research Fishery Biologist
Affiliation: NOAA Fisheries
Email: harvey.walsh@noaa.gov
```

### Project / program

Survey Mitigation

### Data location

Shared google drive, Harvey's personal computer, NCEI

### What is the type of data?

Shapefile [here](https://drive.google.com/drive/folders/1IHYvWZ3o2dPBVSK-TYKDjaqPFcso_Q8P)

### Metadata

- Program description: see InPort or NCEI
- Frequency of update: rarely
- Where is metadata currently stored?
see README (provided by Harvey W) or maybe InPort / NCEI?

### Any data attributes that are non-spatial (e.g., stratum id)?

Yes

### If Yes, please explain

see README file provided from Harvey for data dictionary

### Spatial reference / coordinate system

Geographic Coordinate System: NAD 1983
WKID: 4269
Authority: EPSG
Angular Unit: Degree (0.0174532925199433)
Prime Meridian: Greenwich (0.0)
Datum: D North American 1983
Spheroid:	GRS 1980
Semimajor Axis: 6378137.0
Semiminor Axis: 6356752.314140356
Inverse Flattening: 298.257222101

### Geographic extent

Top: 44.485581 deg
Bottom: 35.142186 deg
Left:	-75.967913 deg
Right: -65.168694 deg

### Measurement system

meters

### General description of project purpose and summary

see InPort for description of EcoMon survey. Harvey will provide if InPort description is not sufficient. 

### Wind developer activity information

Harvey would like access to wind developer scientific survey station information. NOAA NEFSC surveys would like to sample close to developer stations to be able to easily compare collected data.

**Comment by NMucci1** [comment]
[EcoMon public links.docx](https://github.com/user-attachments/files/17835907/EcoMon.public.links.docx)
[EcoMon Strata Map with Fixed Stations.pdf](https://github.com/user-attachments/files/17835909/EcoMon.Strata.Map.with.Fixed.Stations.pdf)
[ReadMe_EcomonStrata_v4b_shapefiles.pdf](https://github.com/user-attachments/files/17835910/ReadMe_EcomonStrata_v4b_shapefiles.pdf)

**Comment by NMucci1** [comment]
[Fixed Oceanography Stations_v11_Nov2024.xlsx](https://github.com/user-attachments/files/18062237/Fixed.Oceanography.Stations_v11_Nov2024.xlsx)

**Comment by jmhatch** [comment]
@NMucci1 The link in this issue points to AMAPPS, could you change it so that so it points to the EcoMon folder? Or wherever the EcoMon strata are stored? Thanks.

**Comment by NMucci1** [comment]
@jmhatch I updated, thanks!

**Comment by jmhatch** [comment]
@NMucci1 Thanks!

**Comment by NMucci1** [comment]
set email to Harvey on 2/11/25 to get feedback on strata on GeoPlatform

**Comment by NMucci1** [comment]
Comments from Harvey: 
-There is an extra s in the title Ecosystem Monitoring (EcoMon) NEFSC**S** scientific survey strata
-I made this change in metadata tables in oracle (I made this mistake in some of the other surveys and corrected all of them) and pushed updates to ArcGIS Online programmatically
-Harvey also sent google doc with metadata updates https://docs.google.com/document/d/1Zy4rMyAGtSG-bi_rl61wo2t86kBAJfP0/edit 
-I made changes based on google doc to metadata table in oracle, pushed changes programmatically
-I asked Harvey about why some survey stations fall outside the strata, Harvey said that the location of some fixed stations outside the strata is on purpose. They want hydrographic data from off the shelf to help understand the transport, etc.

**Comment by jmhatch** [comment]
@NMucci1 I just looked at the EcoMon page and it still has NEFSC**S** in the summary. Not sure if all of the changes made in Oracle are reflected yet?

**Comment by NMucci1** [comment]
@jmhatch the changes should be reflected now! I ran the script this morning.

**Comment by jmhatch** [comment]
@NMucci1 Another quick question. Can the Description property for the layers be the Summary? Seems unnecessary to duplicate the Service's description for each layer. Does this make sense?

**Comment by NMucci1** [comment]
@jmhatch Yes I see what you mean, that makes sense to me. I'll change that.

**Comment by NMucci1** [comment]
put in ticket with NOAA ESRI IT to make feature service public

**Comment by jmhatch** [comment]
Feature server is now public, and data dictionary submission looks good. Nice work! Closing issue.
